mudita/mudita-center
# [CP-1708] handle force sequential update process

Jira: [CP-1708]

**Description**
Scope:
* handling force sequential update process
* feel free to test it
* remove old and unused components (see https://github.com/mudita/mudita-center/pull/1094/commits/4b4f5643b1e36e971f8c098af4cc1fa65e8f5805)

Test details:
* to changes it in the development mode:
    * `packages/app/src/feature-flags/features/index.ts` - change `Feature.ForceUpdate` in `Environment.Development` to true
    * make sure to have Harmony lower than 1.9.0 or MuditaPure lower than 1.6.0 (WARNING: flashing later it to the lower version might be dangerous - see https://appnroll.slack.com/archives/GMVGVEXEW/p1674216960578879?thread_ts=1673342682.681369&cid=GMVGVEXEW) 

<details>
<summary><b>Screenshots</b></summary>
// put images here
</details>


[CP-1708]: https://appnroll.atlassian.net/browse/CP-1708?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
